### ISSUE TITLE
Report if CA2201 is disabled via pragma

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#2194]
## Description
We should report cases where developers suppress [CA2201](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2201) with:

```csharp
#pragma warning disable CA2201
```

We already have `MiKo_3010`, which reports when code throws exceptions that are reserved for the .NET runtime. This is very similar to what `CA2201` checks.

If a developer explicitly suppresses `CA2201`, they knowingly chose to ignore the warning. In most cases, they should have created and thrown a custom exception instead. Because the suppression is intentional, we should consider reporting it.

## Open Questions

- Our own rule can also be disabled or suppressed. Does it still make sense to report `CA2201` suppressions?
- Is the implementation and maintenance effort of a dedicated rule justified for this scenario?
